assets/test_data/samplelist.csv

```diff
@@ -1,2 +1,3 @@
 id,platform,read1,read2
-p1000,illumina,/path/to/JASEN/assets/test_data/sequencing_data/saureus_100k/saureus_large_R1_001.fastq.gz,/path/to/JASEN/assets/test_data/sequencing_data/saureus_100k/saureus_large_R2_001.fastq.gz
```

Comment:
Please let the example config remain as is. Make a seperate config for your local system, or add it to .gitignore